property.go

```diff
@@ -3,15 +3,15 @@ package opslevel
 import "fmt"
 
 type PropertyDefinitionInput struct {
-	Name   string     `json:"name"`
-	Schema JSONString `json:"schema"`
+	Name   string `json:"name"`
+	Schema JSON   `json:"schema"`
```

Comment:
Not sure if this should become `JSONString` and the other should become `JSON`, because of `GetGraphQLType()`.